pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Overview:
+Give a brief overview of the issue you are solving. Succintly explain the GitHub issue you are addressin and the underlying problem of the ticket.
+
+# Addresses:
+Add a link to the issue on Github board
+
+# Type of Fix:
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
Is the user supposed to keep this list in the PR body and just check one?

Reply:
That's the idea. I thought that we could also just reference the github issue but this might be easier at a glance.

Reply:
Sounds good. At the beginning I'd add a comment to clarify. Can just be as simple as : <!-- Check one: -->

Reply:
Added clarifying comment